tests/src/test/java/alluxio/client/fs/FileSystemIntegrationTest.java

```diff
@@ -67,6 +67,9 @@
   private CreateFilePOptions mWriteBoth;
   private UnderFileSystem mUfs;
 
+  private static CreateFilePOptions sWriteBoth = CreateFilePOptions.newBuilder()
```

Comment:
is this used?

Reply:
Whoops. this shouldn't have made it in here